src/StorageSync/StorageSync/help/Reset-AzStorageSyncServerCertificate.md

```diff
@@ -0,0 +1,181 @@
+---
+external help file: Microsoft.Azure.Commands.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storagesync/reset-Azstoragesyncservercertificate
+schema: 2.0.0
+---
+
+# Reset-AzStorageSyncServerCertificate
+
+## SYNOPSIS
+This command will use to reset storage sync server certificate.
+
+## SYNTAX
+
+### ObjectParameterSet (Default)
+```
+Reset-AzStorageSyncServerCertificate [-ParentObject] <PSStorageSyncService> [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### StringParameterSet
+```
+Reset-AzStorageSyncServerCertificate [-ResourceGroupName] <String> [-StorageSyncServiceName] <String>
```

Comment:
StorageSyncServiceName -> Name

Reply:
StorageSyncServer is different than StorageSyncService

---
In reply to: [255695554](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255695554) [](ancestors = 255695554)